hosts

```diff
@@ -9,7 +9,7 @@
 ## +-------------------------  >>d(' _ ')b<<  -------------------------+
 ## |                                                                   |
 ## |         Project: https://github.com/lennylxx/ipv6-hosts           |
-## |         Update : Tue, 31 Oct 2017 13:55:46 +0800                  |
+## |         Update : Fri, 03 Nov 2017 19:07:43 +0800       |

```

Comment:
Seems like you are missing some spaces this line.

Reply:
Change again2333